appveyor.yml

```diff
@@ -0,0 +1,34 @@
+environment:
+  node_pre_gyp_accessKeyId:
+    secure: rQ5epOSve3fmLxaTbCxbHf5msiTpUp385KNMQi0PzXA=
+  node_pre_gyp_secretAccessKey:
+    secure: m0OsuzftpnFSZj6gyyE1pxefcNZVThllytvj/N6fd3Ta8Dx4OtrJ091BU2WhkJ5j
+
+  matrix:
+    - nodejs_version: 0.10.40
+      msvs_toolset: 12
+    - nodejs_version: 0.12.7
+      msvs_toolset: 12
+    - nodejs_version: 1.8.1
+      msvs_toolset: 12
+    - nodejs_version: 2.0.0
+      msvs_toolset: 12
+    - nodejs_version: 3.3.0
+      msvs_toolset: 12
+    - nodejs_version: 4.0.0
+      msvs_toolset: 12
+
+platform:
+  - x86
+  - x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:PLATFORM
+  - cmd: npm install --build-from-source
+
+on_success:
+  - npm run-script publishBinary
+
+build: OFF
+test: OFF
```

Comment:
Appveyor machines don't have bash, so I didn't bother. But it's valid, I agree, so I've added them:
https://github.com/smoogly/png-img/blob/e38bb06c983b986b6120e6782fe35d3fc49dc859/appveyor.yml#L28